packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1974,6 +1977,22 @@ export default class Meeting extends StatelessWebexPlugin {
       this.wirelessShare = true;
     }
 
+    if (options.meetingQuality) {
+      if (typeof options.meetingQuality === 'string') {
```

Comment:
Let's check for a proper string value as well:

```suggestion
      if (typeof options.meetingQuality === 'string' && QUALITY_LEVELS[options.meetingQuality]) {
```